aws_sagemaker_studio/sagemaker_algorithms/linear_learner_mnist/linear_learner_mnist.ipynb

```diff
@@ -86,22 +140,55 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
+   "execution_count": 4,
+   "id": "billion-samoa",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2021-06-14T00:08:01.873326Z",
+     "iopub.status.busy": "2021-06-14T00:08:01.872861Z",
+     "iopub.status.idle": "2021-06-14T00:10:12.008905Z",
+     "shell.execute_reply": "2021-06-14T00:10:12.009298Z"
+    },
+    "papermill": {
+     "duration": 130.152386,
+     "end_time": "2021-06-14T00:10:12.009430",
+     "exception": false,
+     "start_time": "2021-06-14T00:08:01.857044",
+     "status": "completed"
+    },
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "%%time\n",
     "import pickle, gzip, numpy, urllib.request, json\n",
     "\n",
+    "fobj = boto3.client('s3').get_object(\n",
```

Comment:
Not mentioned in the description for changes that were made... not related to pandas...